PWGHF/HFL/Tasks/taskElectronWeakBoson.cxx

```diff
@@ -0,0 +1,248 @@
+#include "EMCALCalib/BadChannelMap.h"
+
+#include "DataFormatsEMCAL/Cell.h"
+#include "DataFormatsEMCAL/Constants.h"
+#include "DataFormatsEMCAL/AnalysisCluster.h"
+
+#include "PWGJE/DataModel/EMCALClusters.h"
+
+using namespace o2;
+using namespace o2::framework;
+using namespace o2::framework::expressions;
+
+struct HfTaskElectronWeakBoson {
+
+  using SelectedClusters = o2::soa::Filtered<o2::aod::EMCALClusters>;
+
+  // PbPb
+  using TrackEle = o2::soa::Filtered<o2::soa::Join<o2::aod::Tracks, o2::aod::FullTracks, o2::aod::TracksExtra_001, o2::aod::TracksDCA, o2::aod::TrackSelection, o2::aod::pidTPCFullEl>>;
+
+  // pp
+  // using TrackEle = o2::soa::Filtered<o2::soa::Join<o2::aod::Tracks, o2::aod::FullTracks, o2::aod::TracksDCA, o2::aod::TrackSelection, o2::aod::pidTPCEl, o2::aod::pidTOFEl>>;
+
+  Preslice<o2::aod::EMCALClusterCells> perCluster = o2::aod::emcalclustercell::emcalclusterId;
+  Preslice<o2::aod::EMCALAmbiguousClusterCells> perClusterAmb = o2::aod::emcalclustercell::emcalambiguousclusterId;
+  PresliceUnsorted<o2::aod::EMCALMatchedTracks> perClusterMatchedTracks = o2::aod::emcalmatchedtrack::trackId;
+
+  Configurable<int> nBinsPt{"nBinsPt", 100, "N bins in pt histo"};
+  Configurable<int> nBinsE{"nBinsE", 100, "N bins in E histo"};
+
+  // event filter
+  Filter eventFilter = (o2::aod::evsel::sel8 == true);
+  Configurable<float> vtxZ{"vtxZ", 10.f, ""};
+  Filter posZFilter = (nabs(o2::aod::collision::posZ) < vtxZ);
+
+  // track cuts
+  Configurable<float> etalow{"etalow", -0.6f, ""};
+  Configurable<float> etaup{"etaup", 0.6f, ""};
+  Configurable<float> dcaxy_cut{"dcaxy_cut", 2.0f, ""};
+  Configurable<float> mimpT_cut{"mimpT_cut", 3.0f, "minimum pT cut"};
+  Configurable<float> itschi2_cut{"itschi2_cut", 15.0f, "its chi2 cut"};
+  Configurable<float> tpcchi2_cut{"tpcchi2_cut", 4.0f, "tpc chi2 cut"};
+  Configurable<float> itsNcl_cut{"itsNcl_cut", 2.0f, "its # of cluster cut"};
+  Configurable<float> tpcNcl_cut{"tpcNcl_cut", 100.0f, "tpc # if cluster cut"};
+  Configurable<float> tpcNclCr_cut{"tpcNclCr_cut", 100.0f, "tpc # of crossedRows cut"};
+
+  Filter filter_globalTr = requireGlobalTrackInFilter();
+  Filter etafilter = (aod::track::eta < etaup) && (aod::track::eta > etalow);
+  Filter dcaxyfilter = (nabs(aod::track::dcaXY) < dcaxy_cut);
+
+  // cluster cut
+  Configurable<int> mClusterDefinition{"mClusterDefinition", 10, "cluster definition to be selected, e.g. 10=kV3Default"};
+  Configurable<float> minTime{"minTime", -25., "Minimum cluster time for time cut"};
+  Configurable<float> maxTime{"maxTime", +20., "Maximum cluster time for time cut"};
+  Configurable<float> minM02{"minM02", 0.1, "Minimum M02 for M02 cut"};
+  Configurable<float> maxM02{"maxM02", 0.9, "Maximum M02 for M02 cut"};
+  Configurable<float> minM20{"minM20", 0.1, "Minimum M20 for M20 cut"};
+  Configurable<float> maxM20{"maxM20", 0.6, "Maximum M20 for M20 cut"};
+  Configurable<float> MatchR_cut{"MatchR_cut", 0.1, "cluster - track matching cut"};
```

Comment:
```suggestion
  Configurable<float> minTime{"minTime", -25., "Minimum cluster time"};
  Configurable<float> maxTime{"maxTime", +20., "Maximum cluster time"};
  Configurable<float> minM02{"minM02", 0.1, "Minimum M02"};
  Configurable<float> maxM02{"maxM02", 0.9, "Maximum M02"};
  Configurable<float> minM20{"minM20", 0.1, "Minimum M20"};
  Configurable<float> maxM20{"maxM20", 0.6, "Maximum M20"};
  Configurable<float> MatchR_cut{"MatchR_cut", 0.1, "cluster - track matching cut"};
```
Do not repeat yourself.
The name of the cut variable and its documentation should be clear enough to tell how the cut is applied. That is not the case for `MatchR_cut`